accord-core/src/test/java/accord/utils/Property.java

```diff
@@ -213,10 +218,11 @@ private static String propertyError(Common<?> input, Throwable cause, Object...
         return sb.toString();
     }
 
-    private static String statefulPropertyError(StatefulBuilder input, Throwable cause, Object state, List<String> history)
+    private static String statefulPropertyError(StatefulBuilder input, Throwable cause, Object state, int failingStep, List<String> history)
     {
         StringBuilder sb = propertyErrorCommon(input, cause);
         sb.append("Steps: ").append(input.steps).append('\n');
+        sb.append("Failing Step: ").append(failingStep).append('\n');
```

Comment:
helpful for larger histories; you can just call the process method with the right debug step and run the test to that point